pycryptobot.py

```diff
@@ -908,8 +901,7 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
 
                 # if not live
                 else:
-                    if app.isVerbose() == 0:
-                        margin, profit, sell_fee = calculate_margin(
+                    margin, profit, sell_fee = calculate_margin(
```

Comment:
Can this scope change cause this error:


 2021-05-21 16:39:00 | USDTTRY 1m | Close: 8.488 | BUY

USDT balance before order: 0.0308
TRY balance before order: 524.3776
Order quantity after rounding and fees: 61.7
USDT balance after order: 61.7308
TRY balance after order: 0.6063
UnboundLocalError("local variable 'margin' referenced before assignment")
Traceback (most recent call last):
  File "/root/bot/USDTTRY/pycryptobot.py", line 1073, in <module>
    main()
  File "/root/bot/USDTTRY/pycryptobot.py", line 1036, in main
    runApp()
  File "/root/bot/USDTTRY/pycryptobot.py", line 1033, in runApp
    s.run()
  File "/usr/lib64/python3.9/sched.py", line 151, in run
    action(*argument, **kwargs)
  File "/root/bot/USDTTRY/pycryptobot.py", line 656, in executeJob
    margin_text = str(app.truncate(margin, 2)) + '%'
UnboundLocalError: local variable 'margin' referenced before assignment